README.md

```diff
@@ -20,12 +20,16 @@ You can install Grafana Logs Drilldown in your own Grafana instance using `grafa
 > The following Loki and Grafana version and configuration are required:
 >
 > - Loki v3.2+
->   - `--pattern-ingester.enabled=true` for pattern ingestion
 >   - Volume endpoint enabled in Loki config:
+>     - `--pattern-ingester.enabled=true` for pattern ingestion
+>   - Level detection enabled in Loki config:      
+>     - `--limits_config.discover_log_levels=true` for pattern ingestion
```

Comment:
Verifying with Trevor that this is the correct CLI syntax

Reply:
I cannot be configure per-tenant via the cli

Reply:
> I cannot be configure per-tenant via the cli
I'm not sure I understand?

Reply:
Oh just saw https://github.com/grafana/logs-drilldown/pull/1143#discussion_r2012441027. Closing this thread